s3torchbenchmarking/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools", "build"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "s3torchbenchmarking"
+version = "0.0.1"
+description = "Tools to run and compare benchmarks against various PyTorch connectors like the s3torchconnector."
+requires-python = ">=3.8,<3.12"
```

Comment:
Does this explicitly require less than 3.12? Ideally I think we'd want to make this as lenient as possible, and take our version restriction from s3torchconnector itself